docs/tutorials/join-dal-baker/run-baker.md

```diff
@@ -36,7 +36,7 @@ You can also refer to [Run a persistent baking node](https://opentezos.com/node-
 
    This command returns all of the topics that the baker is subscribed to in the format `{"slot_index":<index>,"pkh":"<ADDRESS OF BAKER>"}` where `index` varies between `0` included and the number of slot indexes excluded.
 
-   You can also look at the baker logs to see if it injects the expected operations. At each level, the baker is expected to:
+   You can also look at the baker logs to see if it injects the expected operations. At each level, the baker is expected to do a subset of these operations:
```

Comment:
It may take sometime to see some. Something you can do is to grep the daily logs. Today, while debugging Aurélien, we notice it takes about 100 levels to get elected and be assigned to some shard.

Reply:
On weeklynet I get log messages like  this one:

```
Dec 02 19:59:09.054: ready to attach DAL attestation for level 125601, round 0, with bitset 0 for
Dec 02 19:59:09.054:   tz1QCVQinE8iVj1H2fckqx6oiM85CNJSK9Sx to attest slots published at level
Dec 02 19:59:09.054:   125594
```

Reply:
Oh, once you gave me the text to search, I found quite a few in the Octez node log! Thanks.